src/main/java/com/google/sps/data/GameStage.java

```diff
@@ -18,6 +18,8 @@
    * Game Stages are levels on the gameboard. Each level can be accessed by a player, but only one
    * at a time.
    */
+  private static final String DEFAULT_NEXTID_IF_FINAL_STAGE = "noNextId";
+
```

Comment:
Weird line

Reply:
I think the linter wants me to have a line between my static final variables and my other variables.

Reply:
it does